receiver/hostmetricsreceiver/internal/scraper/processscraper/config.go

```diff
@@ -31,6 +31,10 @@ type Config struct {
 	// the collector does not have permission for.
 	MuteProcessIOError bool `mapstructure:"mute_process_io_error,omitempty"`
 
+	// MuteProcessCgroupError is a flag that will mute the error encountered when trying to read the cgroup of a process
+	// the collector does not have permission for.
+	MuteProcessCgroupError bool `mapstructure:"mute_process_cgroup_error,omitempty"`
```

Comment:
This new configuration option should be added to README.

This would be the fifth "mute" option on the process scraper. Perhaps we should add a global "mute_all_errors" option to make life easier for users? 🤔

Reply:
probably good to look at after this PR lands, if it does